deeppavlov/configs/ranking/mltrack_ranker.json

```diff
@@ -0,0 +1,85 @@
+{
+  "dataset_reader": {
+    "class_name": "mltrack_reader",
+    "data_path": "{DOWNLOADS_PATH}/mltrack_data"
+  },
+  "dataset_iterator": {
+    "class_name": "siamese_iterator",
+    "seed": 243
+  },
+  "chainer": {
+    "in": ["x"],
+    "in_y": ["y"],
+    "pipe": [
+      {
+        "class_name": "bert_ranker_preprocessor",
+        "vocab_file": "{DOWNLOADS_PATH}/bert_model/rubert_cased_L-12_H-768_A-12_v1/vocab.txt",
+        "do_lower_case": false,
+        "max_seq_length": 64,
+        "in": ["x"],
+        "out": ["bert_features"]
+      },
+      {
+        "class_name": "bert_ranker",
+        "n_classes": 3,
+        "one_hot_labels": false,
+        "return_probas": false,
+        "bert_config_file": "{DOWNLOADS_PATH}/bert_model/rubert_cased_L-12_H-768_A-12_v1/bert_config.json",
+        "pretrained_bert": "{DOWNLOADS_PATH}/bert_model/rubert_cased_L-12_H-768_A-12_v1/bert_model.ckpt",
+        "save_path": "{MODELS_PATH}/model",
+        "load_path": "{MODELS_PATH}/model",
```

Comment:
There is no model in downloads list. Do you not share a pretrained model?

Reply:
If you mean a model that I was trained using this config, you can find it here in downloads list. I called it fine-tuned because this config allows to fine-tune weights from pretrained bert.  
https://github.com/deepmipt/DeepPavlov/blob/712717a268329015b41de1e60adb84531d1a490c/deeppavlov/configs/ranking/mltrack_ranker.json#L81-L82

Reply:
But the path is different. So you can't just do `model = build_model('mltrack_ranker', download=True)` and get a pretrained model.